Bio/AlignIO/MafIO.py

```diff
@@ -195,9 +199,26 @@ def MafIterator(handle, seq_count=None):
                     )
                 )
             elif line.startswith("i"):
-                # TODO: information about what is in the aligned species DNA before
+                # information about what is in the aligned species DNA before
                 # and after the immediately preceding "s" line
-                pass
+                # i (literal), src (ID), leftStatus, leftCount, rightStatus, rightCount
+                line_split = line.strip().split()
+                # First check that the format is correct
+                if len(line_split) != 6:
+                    raise ValueError(
+                        "Error parsing information - 'i' line must have 7 fields"
+                    )
+                if not records:
+                    raise ValueError("Found 'i' line before sequence of alignment")
+                if records[-1].id != line_split[1]:
+                    raise ValueError(
+                        "Found 'i' line with src different from previous alignment"
+                    )
+                # Add the annotations:
+                records[-1].annotations["leftStatus"] = line_split[2]
+                records[-1].annotations["leftCount"] = int(line_split[3])
+                records[-1].annotations["rightStatus"] = line_split[4]
+                records[-1].annotations["rightCount"] = int(line_split[5])
```

Comment:
If would personally pick something more Pythonic like "right count" but let's see if we get a second opinion before changing that?

Reply:
I would suggest to follow the term definitions in the UCSC specification of the MAF format as closely as possible. Otherwise, we'd have to map the terms used in Biopython to the terms in the MAF file format specification. As UCSC uses leftStatus, leftCount, rightStatus, rightCount, it makes sense to uses these here as well.
For details see
https://genome.ucsc.edu/FAQ/FAQformat.html#format5

Reply:
OK. It isn't like we need to transform the values themselves.